cache.md

```diff
@@ -131,6 +131,27 @@ In contrast, this statement would remove only caches tagged with `authors`, so "
 
 	Cache::tags('authors')->flush();
 
+<a name="cache-events"></a>
+## Cache events
+
+Cache actions can trigger events. You may for instance use `Event` listeners to access the following events.
```

Comment:
`listeners to access the following events.` Seems odd wording. We would nee a colon here for "the following", but there's not one.